### DISCUSSION TITLE
Storing files with Dexie Cloud

### DISCUSSION THREAD

**Discussion by michaelhue** [dexie/Dexie.js#1590] [Q&A] [answered]
Hi! Is there anything one should be aware of when storing files/blobs with Dexie Cloud (performance, size limits, etc.)?

**Comment by dfahlander** [comment, marked as answer]
Thanks for a good question. Blobs are currently stored as base64 strings within a JSON field in the database. Right now we don't yet put any limitation on its size but may be having to do that unless we decide to store large blobs on alternative storage in a future version. I would currently not recommend storing larger blobs than 10 MB in dexie cloud.

**Reply by michaelhue** [reply]
Thanks David, sounds good! My use case would involve photos in JPEG format, so the limit should not be a problem. Using something like [image-blob-reduce](https://github.com/nodeca/image-blob-reduce) could also be an option to resize large images on the client side.
